config/application.rb

```diff
@@ -2,9 +2,9 @@
 
 require "rails"
 # Pick the frameworks you want:
-# require "active_model/railtie"
+require "active_model/railtie"
```

Comment:
Fairly sure you don't need this when you have active record, and in any case this app has no forms.